services/flows-operator/src/FlowOperator.js

```diff
@@ -256,20 +256,39 @@ class FlowOperator {
                     spec: {
                         restartPolicy: 'Never',
                         containers: [{
-                            image: node.image,
-                            name: 'apprunner',
-                            imagePullPolicy: 'Always',
-                            env: Object.keys(envVars).map(key => ({
-                                name: key,
-                                value: envVars[key]
-                            }))
+                          image: node.image,
```

Comment:
AFAIK  we are using 4-spaces indent. Why only two here?

Reply:
sorry, my IDE was misconfigured